base/context.go

```diff
@@ -46,6 +47,9 @@ type Context struct {
 	// Required unless Insecure is true.
 	Certs string
 
+	// KeySize is the key size in bits for certificate keys.
+	KeySize int
```

Comment:
Isn't this only required on cert generation? Seems strange to be placing it in `base.Context`.